1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -490,3 +494,17 @@ const quotes = [
     author: "Zig Ziglar",
   },
 ];
+
+let quoteTitle = document.getElementById("quote");
+let authorName = document.getElementById("author");
+let button = document.getElementById("new-quote");
+
+function getQuotes() {
+  let randomQuote = pickFromArray(quotes);
+  quoteTitle.innerHTML = `<span class="quote-symbol">&ldquo;</span> ${randomQuote.quote}`;
```

Comment:
Nice styling for the quote.
Btw If it were me adding the styled `"` before the quote, I would put that html into index.html file, and keep the js file in js code only. because if someone want change the layout they can just go to the index.html file, easier to maintain.
we can change the html like below, then we can add the text to "quote-content".
```
<h2 id="quote" class="quote">
  <span class="quote-symbol">&ldquo;</span>
  <span id="quote-content"></span> 
</h2>
```